views/overview/main_panel.go

```diff
@@ -123,3 +154,28 @@ func (p *MainPanel) refreshWorkloadSummary(ctx context.Context, summary model.Cl
 	}
 	return nil
 }
+
+// filterColumns filters the allColumns based on the user-provided filterCols
+// It returns a slice of columns that match the case-insensitive filter
+func filterColumns(allColumns []string, filterCols []string) []string {
```

Comment:
This func probably can be simplified using the the `slice` package (i.e. `slice.Contains`)